entries/s/shy.gg

```diff
@@ -0,0 +1,17 @@
+{
+  "Shy": {
+    "domain": "shy.gg",
+    "img": "shy.gg.svg",
+    "additional-domains": [
+      "past.rip",
+      "weak.gg",
+    ],
+    "tfa": [
+        "totp"
+    ],
+    "documentation": "https://shy.gg/app/account",
```

Comment:
```suggestion
```
The documentation needs to publically accessible (without logging in), so this also isn't needed.